db/migrate/20160225180013_create_items.rb

```diff
@@ -0,0 +1,8 @@
+class CreateItems < ActiveRecord::Migration
+  def change
+    create_table :items do |t|
```

Comment:
This is me leaving you a kind, specific, and actionable comment, to prove that I can. This line is awesome.